freud/density.pyx

```diff
@@ -508,12 +509,19 @@ cdef class RDF(SpatialHistogram1D):
 
     @Compute._computed_property
     def RDF(self):
+        """(:math:`N_{bins}`,) :class:`numpy.ndarray`: Histogram of RDF
+        values."""
         return freud.util.make_managed_numpy_array(
             &self.thisptr.getRDF(),
             freud.util.arr_type_t.FLOAT)
 
     @Compute._computed_property
     def n_r(self):
+        """(:math:`N_{bins}`,) :class:`numpy.ndarray`: Histogram of cumulative
+        bin_counts values. More precisely, :code:`n_r[i]` is the average number
+        of points contained within a ball of radius :code:`R[i]+dr/2` centered
+        at a given :code:`query_point` averaged over all :code:`query_points`
+        in the last call to :meth:`~.compute` (or :meth:`~.accumulate`)."""
```

Comment:
How to write this without accumulation?
```suggestion
        computed thus far."""
```

Reply:
That seems about as good as we can do...